.github/workflows/main.yml

```diff
@@ -91,13 +99,18 @@ jobs:
           path: |
             ~/.cache/bazel
             ~/.cache/bazel-repo
-          key: v4-bazel-repo-cache-${{ matrix.toolchain }}-${{ hashFiles('.bazelversion', 'examples/WORKSPACE', 'repositories/**', 'requirements_lock.txt', 'WORKSPACE') }}
-          restore-keys: v4-bazel-repo-cache-${{ matrix.toolchain }}-
-      - name: bazel test ${{ matrix.config_option }} //...
```

Comment:
It helps me to visually distinguish builds by having config_option in the name. I'd keep this one.

Reply:
updated